worker-markup-container-fs/pom.xml

```diff
@@ -297,7 +297,7 @@
                             <alias>worker-markup-fs</alias>
                             <name>${dockerDataProcessingOrg}worker-markup${dockerProjectVersion}</name>
                             <build>
-                                <from>cafapi/opensuse-jeptalon:1</from>
+                                <from>cafinternal/prereleases:opensuse-jeptalon-1.2.0-SCMOD-6825-SNAPSHOT</from>
```

Comment:
Don't forget to remove branch before merging.

Reply:
Removed branch build version